locations/spiders/c_and_a.py

```diff
@@ -74,7 +76,7 @@ def parse_city(self, response):
                 "name": store.xpath('./div[@class="addressBox"]/p[@class="store"]/text()').get(),
                 "phone": phone.replace("Tel: ", "") if phone else None,
                 "country": country,
-                "opening_hours": "; ".join(hours),
+                "opening_hours": opening_hours.as_opening_hours(),
```

Comment:
This is fine as is, but just so you know you can use the `OpeningHours` object directly and we'll call `as_opening_hours()` before writing to output.